en_US.json

```diff
@@ -1538,7 +1538,21 @@
 	"property.origin.closeButton": "Close",
 	"property.origin.dialog.name": "Origin Details",
 	"property.origin.editButton": "Save",
+	"property.origin.name.always.direct": "Always Direct",
+	"property.origin.name.auto": "Auto",
+	"property.origin.name.both": "Both",
+	"property.origin.name.false": "False",
+	"property.origin.name.link.validate": "Validate",
+	"property.origin.name.no": "No",
+	"property.origin.name.no.direct": "No Direct",
+	"property.origin.name.none": "None",
+	"property.origin.name.off": "Off",
+	"property.origin.name.on": "On",
 	"property.origin.name.placeholder": "Enter a name for your origin.",
+	"property.origin.name.remove": "Remove",
+	"property.origin.name.true": "True",
+	"property.origin.name.validate.confirm.text": "Please enter server URL to validate.",
```

Comment:
I suggest  "Please enter a server URL to validate."
In what situation is this message shown?  I had trouble getting it to appear in the portal.

Reply:
we can see popup on the 'validate' link when mouse over on it if server is empty

Reply:
@jungho73  In this case, I think it is more appropriate to say
"Please enter a server to validate."

The user is prompted to enter a hostname or IP address in the text box and optionally alternate port numbers.

A URL is like http://domain.com.
Though we may construct a URL to validate against based on the server and IP addresses entered, the user is not technically entering a "URL".